demuxer_test.go

```diff
@@ -84,6 +94,37 @@ func TestDemuxerNextData(t *testing.T) {
 	assert.EqualError(t, err, ErrNoMorePackets.Error())
 }
 
+func TestDemuxerNextDataPATPMT(t *testing.T) {
+	pat := hexToBytes(`474000100000b00d0001c100000001f0002ab104b2ffffffffffffffff
```

Comment:
How do you come up with those hexes though?

Reply:
I am getting these directly from real world mpegts samples, using a combination of head/tail and xxd:

>tail -c +$((1+188*5422)) input.ts | head -c 188 | xxd -p 

To get the packet offset (i.e 5422) I use DVBInspector or tstools/tsinfo to inspect the stream and find the relevant packets (PAT and PMT in this case).